kani-compiler/src/codegen_cprover_gotoc/codegen/static_var.rs

```diff
@@ -37,9 +36,8 @@ impl<'tcx> GotocCtx<'tcx> {
         // havoc static variables. Kani uses the location and pretty name to identify
         // the correct variables. If the wrong name is used, CBMC may fail silently.
         // More details at https://github.com/diffblue/cbmc/issues/8225.
```

Comment:
Is it the right place for that comment ?  shouldn't it be moved to where `--nondet-static-exclude` is actually used ?

```suggestion
        // CBMC contract instrumentation havocs all statics by default (for soundness). 
        // However, some static variables added by Kani's own contract instrumentation must not be havoced
        // in order for the instrumentation to work as intented (if these variables get havoced,
        // the analysis may fail spuriously, cf.  https://github.com/diffblue/cbmc/issues/8225).
        // Kani uses location and pretty name to identify these variables and uses 
        // `--nondet-static-exclude` to exclude these variables from havocing.
```

Reply:
I didn't change this code, but we can update the doc. No problem.